boggle.js

```diff
@@ -29,37 +29,38 @@ function printBoggleBoard(boggleBoard) {
 //   console.log(allArrays);
 // }
 
-function randomRange(length) {
-  return Math.floor(Math.random() * length - 1);
+function randomRange(arrayLength) {
+  return Math.floor(Math.random() * arrayLength);
 }
 
-function removeElement(allArrays, index) {
-  let item = allArrays.splice(index, 1)[0];
+function removeElement(selectedDice, index) {
+  let item = selectedDice.splice(index, 1)[0];
   return item;
 }
 
-function shuffle(allArrays) {
-  while (allArrays.length > 0) {
-    let randIndex = randomRange(allArrays.length);
-    let removedElem = removeElement(allArrays, randIndex);
-    shuffledArray.push(removedElem);
-   // return shuffledArray;
+function shuffle(selectedDice) {
+  while (dice.length > 0) {
+  // console.log(dice);
+    let randIndex = randomRange(dice.length);
+    let removedElem = removeElement(dice, randIndex);
```

Comment:
Try shuffling the dice twice and see what happens. You'll be surprised, I think.